spec.md

```diff
@@ -107,27 +107,19 @@ the event data. The context might also need to be serialized with the event
 data for some use cases (e.g. a JSON implementation might use one JSON object
 that contains both context and data).
 
-### namespace
-* Type: String
-* Description: Identifier that uniquely identifies the organization publishing
-  the event.
-* Constraints:
-  * REQUIRED
-  * MUST be a non-empty string
-* Examples:
-  * kafka.apache.org
-  * com.microsoft.azure
-
 ### event-type
 * Type: String
-* Description: Type of the event `data`. Producers can specify the format of
-  this, depending on their service. This enables the interpretation of `data`,
-  and can be used for routing, policy and more.
+* Description: Type of occurrence which has happened. The event type MUST be
+  namespaced with a package based on the reverse-DNS of a domain associated
+  with the software that produced the event. This MAY be used for routing,
+  observability, policy enforcement, etc.
 * Constraints:
   * REQUIRED
   * MUST be a non-empty string
+  * MUST be prefixed with a reverse-DNS name associated with the software that
+         produces the event
 * Examples:
-  * customer.created
+  * com.github.pull.create
```

Comment:
nitpicking: `com.github.pull.create` reads as a command rather than an event that took place.

Reply:
Note: [github docs](https://developer.github.com/webhooks/#events) use "create" for a notification that something has been created... not that we need to conform to some specific vendor.  We could include a few divergent examples to be clear that we're not mandating anything or consider some guidance about naming if someone wants to see if there's a trend in existing implementations or other spec we want to borrow from.

Reply:
agree this is a nit, and suggest it could be addressed in a follow-on PR

Reply:
As a consumer of an event, "event-type", to me, denotes whether it is a storage event or an messaging event or a timer event , etc.  Then depending on the type of event, it could have different properties. For example, a storage event could have properties like "bucket name", "action performed on that bucket (created or update or delete)", a messaging event could have properties like "topic of the message".

Reply:
@SeanFeldman I lean 20% more towards infinitive tense for a few reasons:

1. It's nice that these names can be reused for other policies in the source software. E.g. only some developers will have "com.github.pull.create" permission in a repository.
2. There seems to be precedent today for events to be named in the infinitive. E.g. `onClick` rather than `onClicked`

Reply:
First scenario is describing a permision, not an event. Permission to **create** a PR.
Second example is historical `OnClick` as an event handler for the `click` event. You'd not call it "WhenClick", but rather "WhenClicked" if that would be the convention back in the day.

I might remain in the minority with my belief that an event is a manifistation of something immutable that happened and therefore should be described in the past tense 🙂